src/commands/epgsql_cmd_connect.erl

```diff
@@ -277,3 +278,14 @@ hex(Bin) ->
                (N) when N < 16 -> $W + N
             end,
     <<<<(HChar(H)), (HChar(L))>> || <<H:4, L:4>> <= Bin>>.
+
+deadline(Timeout) ->
+    erlang:monotonic_time(millisecond) + Timeout.
+
+timeout(Deadline) ->
+    case (Deadline - erlang:monotonic_time(millisecond)) of
```

Comment:
Just noticed CI failure on OTP-18. Seems we'll need to use deprecated `milli_seconds` time unit.